src/makeCalculator.js

```diff
@@ -4,7 +4,32 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    add: (num) => result + num,
+
+    subtract: (num) => result - num,
+
+    multiply: (num) => result * num,
+
+    divide: (num) => (num !== 0 ? result / num : NaN),
+
+    operate(callback, num) {
+      result = callback(num);
```

Comment:
The `operate` method should apply the callback to the current `result` and the `num` parameter. Consider changing it to `result = callback(result, num);` to ensure the operation is applied correctly.